pubsub/azure/servicebus/queues/servicebus.go

```diff
@@ -76,7 +76,7 @@ func (a *azureServiceBus) Publish(req *pubsub.PublishRequest) error {
 	ebo := backoff.NewExponentialBackOff()
 	ebo.InitialInterval = time.Duration(a.metadata.PublishInitialRetryIntervalInMs) * time.Millisecond
 	bo := backoff.WithMaxRetries(ebo, uint64(a.metadata.PublishMaxRetries))
-	bo = backoff.WithContext(bo, a.publishCtx)
+	bo = backoff.WithContext(bo, ctx)
```

Comment:
Here too, please delete the `publishCtx` property from the struct, which is now unused

Reply:
Done.